src/app/wallets/add-invoice-for-wallet.ts

```diff
@@ -16,6 +16,22 @@ import { LndService } from "@services/lnd"
 import { WalletInvoicesRepository, WalletsRepository } from "@services/mongoose"
 import { RedisRateLimitService } from "@services/rate-limit"
 
+export const addInvoiceByWalletPublicId = async ({
```

Comment:
currently invoices are still be created by `user.id`?

Reply:
Yup current code is this ([here](https://github.com/GaloyMoney/galoy/pull/746/files/5a79ed3824b0546e7b23311716a5e889a0fac87b#diff-db3c5b3e8b3f24796d542d4e5c92a62bf101618085c4776bd53c2e0124d4cfc4L32))